packages/core/src/Cardano/types/ProtocolParameters.ts

```diff
@@ -110,7 +110,7 @@ type BabbageProtocolParameters = Omit<AlonzoProtocolParams, 'coinsPerUtxoWord' |
 export interface PoolVotingThresholds {
   motionNoConfidence: Fraction;
   committeeNormal: Fraction;
-  commiteeNoConfidence: Fraction;
```

Comment:
This will hurt on merge to `conway-era` 😅 
https://github.com/input-output-hk/cardano-js-sdk/commit/127dd03e41ba393240d67c794acb2b02163c5113